quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/FeaturesDistTest.java

```diff
@@ -55,9 +55,7 @@ public void testEnablePreviewFeatures(LaunchResult result) {
     public void testEnablePreviewFeaturesAndFips(LaunchResult result) {
```

Comment:
Does this test still makes sense?

Reply:
Yes, I think it still makes sense. It tests that it is possible to enable "preview" features together with "fips" feature at the same time, which is still valid use-case IMO.